templates/common/.github/workflows/bicep/azure-dev.yml

```
@@ -19,9 +19,16 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Log in with Azure
-        uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
+        run: |
+          $info = $Env:AZURE_CREDENTIALS | ConvertFrom-Json -AsHashtable;
+
+          azd login `
+            --client-id "$info.clientId" `
+            --client-secret "$info.clientSecret" `
+            --tenant-id "$info.clientSecret"
```

I'm guessing this is wrong?

```suggestion
            --tenant-id "$info.tenantId"
```

---

The one part of this I had not tested end to end - but yes this is wrong.  Thanks for the fix.